File: test/fixtures/python/post_json.py
import requests

headers = {
    'Content-Type': 'application/json',
    'Accept': 'application/json',
}

json_data = {
    'drink': 'coffe',
}

response = requests.post('http://localhost:28139', headers=headers, json=json_data)

# Note: json_data will not be serialized by requests
# exactly as it was in the original request.
#data = '{ "drink": "coffe" }'
#response = requests.post('http://localhost:28139', headers=headers, data=data)

File: test/fixtures/node/post_json.js
import fetch from 'node-fetch';

fetch('http://localhost:28139', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'Accept': 'application/json'
  },
  // body: '{ "drink": "coffe" }',
  body: JSON.stringify({
    'drink': 'coffe'
  })
});
